README.md

```diff
@@ -4,7 +4,7 @@ Welcome to Bottlerocket!
 
 Bottlerocket is a free and open-source Linux-based operating system meant for hosting containers.
 
-If you’re ready to jump right in, read one of our setup guides for running Bottlerocket in [Amazon EKS](QUICKSTART-EKS.md), [Amazon ECS](QUICKSTART-ECS.md), or [VMware](QUICKSTART-VMWARE.md).
+If you’re ready to jump right in, read one of our setup guides for running Bottlerocket in [Amazon EKS](QUICKSTART-EKS.md), [Amazon ECS](QUICKSTART-ECS.md), [VMware](QUICKSTART-VMWARE.md), or provisioning on [bare metal](PROVISIONING-METAL).
```

Comment:
I would move the provisioning note to its own sentence - it's much harder to jump into.

Something like: "If you're interested in running Bottlerocket on bare metal servers, please refer to the provisioning guide to get started."